src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -452,14 +454,14 @@ void WriteLog<I>::append_op_log_entries(GenericLogOperations &ops) {
       }
   });
   // Append logs and update first_free_update
-  uint64_t bytes_allocated_updated;
+  uint64_t bytes_allocated_updated = 0;
   append_ops(ops, append_ctx, new_first_free_entry, bytes_allocated_updated);
```

Comment:
Which uninitialized variable are you referring to the initial comment?

I wanted to patch it for my local tests so that there is one less bug to run into and thought it was this one, but it is initialized at the top of `append_ops()` and you just moved that initialization here.

Reply:
You maybe right, I must've missed that I removed the variable in `append_ops` and added it back up here when I encountered a bug.

I've marked and commented on a couple of variables that I had changed, incase you want to patch this change.